ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py

```diff
@@ -420,6 +422,13 @@
           - This parameter is only valid with the C(http_header) type.
         type: str
         version_added: "1.8.0"
+      method_matches_with_any:
+        description:
+          - A list of strings of characters the HTTP Method value should match.
+          - This parameter is only valid with the C(http_method) type.
+        type: list
+        elements: str
+        version_added: "1.9.1"
```

Comment:
its not version 1.9.1 it will be version 1.10.0

Reply:
changed